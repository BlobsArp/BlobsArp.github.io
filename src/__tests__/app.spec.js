import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from '../app.jsx';

it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
});
