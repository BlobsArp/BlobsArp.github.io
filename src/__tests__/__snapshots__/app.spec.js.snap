// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App--Container"
  >
    Welcome to React Boilerplate!
  </div>
</DocumentFragment>
`;
